.changeset/tough-forks-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'vercel': patch
+---
+
+Handle requests for deployment aliases when mocking deployments
```

Comment:
```suggestion
---
---

```

Since this is changing tests only, it doesn't require any version change to semver. 

You can do this with an empty changeset.